blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupBatchWriteRunnable.java

```diff
@@ -51,11 +53,12 @@ public void run() {
         } catch (Exception e) {
             LOG.warn("not able to insert rollups", e);
             executionContext.markUnsuccessful(e);
-
+        } finally {
+            executionContext.decrementWriteCounter(writeContexts.size());
+            rollupsPerBatch.update(writeContexts.size());
+            rollupsWriteRate.mark(writeContexts.size());
```

Comment:
@ChandraAddala , I think they serve different purpose. 

The new Meter I added: rollupsWriteRate, measures the rate of locators write per minute. In the dashboard, I am graphing the m1_rate. This is essentially our Rollup Write throughput. 
http://graphite00.dfw.perf01.bf.k1k.me/grafana/#/dashboard/db/rollup-dashboard?panelId=14&fullscreen&from=now-4d&to=now

The old Histogram rollupsPerBatch, does not let me get the m1_rate, since it's a Histogram. We can plot mean, max, p95, p99 from this metric:
http://graphite00.dfw.perf01.bf.k1k.me/grafana/#/dashboard/db/rollup-dashboard?panelId=38&fullscreen&from=now-4d&to=now

The Timer measures the time of this block. It does let you graph m1_rate, but it's the rate of the method call. There is no way for me to add the size of the list. 

Out of the 3 metrics, I'd say I need the Meter and the Timer the most. The Histogram doesn't provide me with too much info, although I think it's useful too.

Do you think I should remove the Histogram?